common/math/math.go

```diff
@@ -42,10 +46,34 @@ func CalculatePercentageGainOrLoss(priceNow, priceThen float64) float64 {
 	return (priceNow - priceThen) / priceThen * 100
 }
 
-// CalculatePercentageDifference returns the percentage of difference between
-// multiple time periods
-func CalculatePercentageDifference(amount, secondAmount float64) float64 {
-	return (amount - secondAmount) / ((amount + secondAmount) / 2) * 100
+// CalculatePercentageDifference returns the percentage difference between two
+// numbers
+func CalculatePercentageDifference(a, b float64) float64 {
```

Comment:
`GetPercentageDifferenceAggressively()` ? 😆 What do you suggest for the name?

Reply:
Generally Get or Fetch should be for something you need to go and find, where it might not be obvious that you're going external to get it. Most of the time I feel they should be left out as well though.
They all talk too much about the methodology, which isn't relevant (and in some cases might change, when caching is introduced).

So just `PercentageDifference` or `PctDiff`.
I'd probably say `PercentageDelta`.

Reply:
`PercentageDifference` is good 👍

Reply:
Part of the rename can change `CalculatePercentageGainOrLoss` to `PercentageChange` to make it less specific. Then rename all funcs to have their function's purpose first. So:

`PercentageChange` > native
`PercentageChangeDecimal` > decimal etc.

I think this would be more useful for people using autocomplete features in their IDE since ATM we have a lot of prefixed `DecimalX`'s which could spam the user with options

Reply:
[here ya go](https://github.com/thrasher-corp/gocryptotrader/pull/1617/commits/7d6e80c6558b3c8871641512ac9fbf9fa1758586)